_posts/2023-08-29-what-in-the-ml-is-going-on-around-here.md

```diff
@@ -0,0 +1,473 @@
+---
+date: 2023-08-30 00:00:00 -0700
+meta_description: getting started with the neural search plugin for opensearch
+meta_keywords: neural search, guide, getting started
```

Comment:
Updated meta: 
**meta_keywords:** Neural search plugin, machine learning OpenSearch, get started with neural search, k-NN index
**meta description:** Learn how to get started with the neural search plugin for OpenSearch to setup a
k-NN index associated with a neural search pipeline and an ML model.